rlkit/data_management/offline_data_store.py

```diff
@@ -0,0 +1,37 @@
+"""
+Class for storing data.
+"""
+import numpy as np
+
+
+class OfflineDataStore(object):
```

Comment:
should this not be attached to a kind of replay buffer? We have a lot of different types of replay buffer in this code and it might be good to keep that interface?